subsys/esb/esb.c

```diff
@@ -941,26 +1029,9 @@ static void get_and_clear_irqs(uint32_t *interrupts)
 
 static void radio_irq_handler(void)
 {
-	if (NRF_RADIO->EVENTS_READY &&
-	    (NRF_RADIO->INTENSET & RADIO_INTENSET_READY_Msk)) {
-		NRF_RADIO->EVENTS_READY = 0;
-		ESB_SYS_TIMER->TASKS_START;
-	}
-
-	if (NRF_RADIO->EVENTS_END &&
-	    (NRF_RADIO->INTENSET & RADIO_INTENSET_END_Msk)) {
-		NRF_RADIO->EVENTS_END = 0;
-		/* Call the correct on_radio_end function, depending on the
-		 * current protocol state.
-		 */
-		if (on_radio_end) {
-			on_radio_end();
-		}
-	}
```

Comment:
These two paths have been removed, but it's not clear why. Were they not necessary?

Reply:
They were not needed. The END interrupt was never enabled and `on_radio_end` pointer was not set to any function.
The READY event looks like duplication to the PPI connection and timer start call on Tx transaction end. I removed interrupt and ppi connection